streaming/src/test/scala/org/apache/spark/streaming/MapWithStateSuite.scala

```diff
@@ -39,18 +39,15 @@ class MapWithStateSuite extends SparkFunSuite
 
   before {
     StreamingContext.getActive().foreach { _.stop(stopSparkContext = false) }
```

Comment:
Just noticed. I think 

```scala
foreach(_.stop(stopSparkContext = false))
```
might be better although this is not the part of this PR.